rfcs/react-components/convergence/semantic-shared-colors.md

```diff
@@ -0,0 +1,195 @@
+| `lightGreen` |         | available |         |
+| `marigold`   |         | away      | persona |
+
+As you can see in the table above, there is an overlap between the status and persona groups - `red` and `marigold` colors from shared color tokens are used as persona shared colors as well. That is intentional, for that reasons, the "persona slots" have the same color mapping in both groups:
+
+| Color slot                          | Notes                 |
+| ----------------------------------- | --------------------- |
+| `colorPaletteRedBackground1`        | status                |
+| `colorPaletteRedBackground2`        | status + persona      |
+| `colorPaletteRedBackground3`        | status                |
+| `colorPaletteRedForeground1`        | status                |
+| `colorPaletteRedForeground2`        | status + persona      |
+| `colorPaletteRedForeground3`        | status                |
+| `colorPaletteRedForegroundInverted` | Red, Green and Yellow |
+| `colorPaletteRedBorderActive`       | status + persona      |
+| `colorPaletteRedBorder1`            | status                |
+| `colorPaletteRedBorder2`            | status                |
+
+This results in 150 (28 _ 3 + 7 _ 9 + 3) color tokens.
+
+Adding new tokens comes at a [runtime performance cost](https://github.com/microsoft/fluentui/blob/master/rfcs/react-components/convergence/theme-shape.md).
+
+## Problem statement
+
+There is no semantic meaning in the current shared color tokens. The intended use case is communicated only by a design spec. See [#26168](https://github.com/microsoft/fluentui/issues/26168), [#25720](https://github.com/microsoft/fluentui/issues/25720).
+Designers plan to update the color values for the status shared colors - that might seem just like a hex value change, but that results in a color name change: for example **red** will be replaced by **cranberry**.
+
+### Token change request from design
+
+The current requirement from design is to keep **presence** and **persona** token names and introduce semantic names for **status** tokens. This is the ideal end state:
+
+| Current name | Current meaning       | Proposed change                                                                                                                     |
+| ------------ | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `red`        | danger, busy, persona | Introduce semantic `danger`, mapped to `cranberry` instead of `red`.<br/>Keep `red`, it is still used for **busy** and **persona**. |
+| `green`      | success               | Replace by semantic `success`.                                                                                                      |
+| `darkOrange` | severe                | **severe** should not be used anywhere. Keep `darkOrange` (temporarily) for Teams compatibility.                                    |
```

Comment:
if it shouldn't be used, why is it here? Is it only for teams compat, and if so, what is the plan to deprecate and remove it?